### ISSUE TITLE
nuclide-start-server, npm permissions and $PATH

### ISSUE THREAD

**Issue by get-trucked** [facebook/nuclide#522]
This is not really a bug but most likely a documentation issue that may need some attention.

So I was getting the nuclide-start-server not found, Now after trawling through the code and getting it to execute a echo $PATH instead of the start server command the non interactive shell is missing the path to execute.

After following npm [documentation](https://docs.npmjs.com/getting-started/fixing-npm-permissions#option-2-change-npms-default-directory-to-another-directory) to fix permissions by not using sudo to install global packages, It tells you add the export path to .profile

Therefore in the .bashrc it does not execute .profile as its a non interactive terminal that is being used by nuclide and that is where the problem lies.
This is in the top of the .bashrc 

` # If not running interactively, don't do anything
case $- in
    *i*) ;;
      *) return;;
esac
`

Logging in via interactive terminal it shows the path added into .profile
 ssh hello@192.168.2.108
```
hello@hello-VirtualBox:~$ echo $PATH
/home/hello/.npm-global/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
```

### How to fix
The export path that is mentioned in the npm documentation must be added to the top of the .bashrc before the non interactive return command mentioned above.

### Expected Behavior
nuclide server to start

### Actual Behavior
Error in Atom
`Unexpected error occurred: bash: nuclide-start-server: command not found .
Original error message:
 bash: nuclide-start-server: command not found
`

### Versions
* Atom: 1.7.3
* Nuclide: 0.139.0 (both client and server)
* Client OS: Linux Mint 17
* Server OS (optional): Ubuntu 16.04

### Additional Details, prob unecessary but here form complteness
apm ls --installed output:
```
Community Packages (9) /home/sypro/.atom/packages
├── autocomplete-python@1.7.3
├── highlight-selected@0.11.2
├── jshint@1.8.5
├── linter@1.11.4
├── linter-pep8@1.3.0
├── linter-pylint@1.2.0
├── nuclide@0.139.0
├── python-indent@0.4.3
└── python-tools@0.6.7

└── (empty)
```

**Comment by nmote** [comment]
It's probably worth noting this somewhere, but not everybody has that line at the top of their .bashrc. Some generic troubleshooting steps, at the very least, would be helpful.

**Comment by get-trucked** [comment]
Yes good point, this is more aimed at debian/ubuntu users as it is in the default .bashrc as far as I know

**Comment by lykosis** [comment]
Thanks for sharing, I had the same issue using these versions:

    Atom: 1.16.0
    Nuclide: 0.217.0 (both client and server)
    Client OS: Mac OS Sierra 10.12.3
    Server OS (optional): Ubuntu 16.04
